Collatz.java

```diff
@@ -1,6 +1,31 @@
 // Demonstrates the Collatz conjecture.
 public class Collatz {
 	public static void main(String args[]) {
-	    // Replace this comment with your code
+	    int num = Integer.parseInt(args[0]);
+		String cOrV = args[1];
+		int insideLoopCouter = 0;
+		int outsideLoopCounter = 0;
+		int newNum;
```

Comment:
It is best practice not to declare a variable without setting a value to it...
(In general - try thinking about what is the value you want this variable to have in case your code does not change it's value)